subprojects/ide/src/main/java/org/gradle/plugins/ide/internal/tooling/EclipseModelBuilder.java

```diff
@@ -197,53 +199,72 @@ private void populate(Project project) {
         }
 
         DefaultEclipseProject eclipseProject = findEclipseProject(project);
+
         eclipseProject.setClasspath(externalDependencies);
         eclipseProject.setProjectDependencies(projectDependencies);
         eclipseProject.setSourceDirectories(sourceDirectories);
+        eclipseProject.setClasspathContainers(classpathContainers);
+        eclipseProject.setOutputLocation(outputLocation != null ? outputLocation : new DefaultEclipseOutputLocation("bin"));
 
-        List<DefaultEclipseLinkedResource> linkedResources = new LinkedList<DefaultEclipseLinkedResource>();
-        for (Link r : eclipseModel.getProject().getLinkedResources()) {
-            linkedResources.add(new DefaultEclipseLinkedResource(r.getName(), r.getType(), r.getLocation(), r.getLocationUri()));
+        // The new Project means 'beforeMerged' won't get to see any previous state from the .project file.
```

Comment:
Yeah, it was a bit wordy. Can I interest you in this one final line? ;-) Fine if not.

Reply:
❌ IMO the comment adds more confusion than clarification. Let's get rid of it.